libs/dh/shared/environments/src/lib/app-environment/dh-app-environment.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { InjectionToken } from '@angular/core';
+import { dhLocalAppEnvironment } from '@energinet-datahub/dh/shared/assets';
+
+import { environment } from '../environment';
+
+export interface DhAppEnvironmentConfig {
+  current: DhAppEnvironments;
+}
+
+export enum DhAppEnvironments {
+  local = 'localhost',
+  dev = 'u-001',
+  test = 't-001',
+  prod = 'b-002',
```

Comment:
suggestion: it'd be nice to still include `b-001` (preProd) in this list but don't expose it publicly. So either using a TypeScript util or maybe having two lists (one exposed and one internal)?